__docs__/wonder-blocks-icon/accessibility.mdx

```diff
@@ -13,19 +13,19 @@ import * as AccessibilityStories from './accessibility.stories';
 * The color contrast should pass WCAG.
   * At least 3:1 contrast ratio for icons.
 * Ideally, an icon should be accompanied by text. However, if an icon
-  is not accompanied by text, provide alt text.
```

Comment:
I replaced references to "alt text" with "aria-label" since the [PhosphorIcon component](https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-icon/src/components/phosphor-icon.tsx#L13) has an `aria-label` prop instead of an `alt` prop. Please let me know though if I'm missing some context!